projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/torch_ods_gen.py

```diff
@@ -424,6 +424,12 @@ def emit_with_mutating_variants(key, **kwargs):
     emit(
         "aten::batch_norm : (Tensor, Tensor?, Tensor?, Tensor?, Tensor?, bool, float, float, bool) -> (Tensor)"
     )
+    emit(
+        "aten::instance_norm : (Tensor, Tensor?, Tensor?, Tensor?, Tensor?, bool, float, float, bool) -> (Tensor)"
```

Comment:
I don't see any patterns for this op. Do we need it in this PR?

Reply:
no